### DISCUSSION TITLE
3D Printed Case for Hardware_tiny

### DISCUSSION THREAD

**Discussion by mptr8956** [JanM321/esphome-lg-controller#48] [Show and tell]
As some folks asked for a case, i designed a very simple one for 3d printing at home. See pictures. Printed without top/bottom shell layers. Not perfect, but good enough to protect the device in my false ceiling and getting enough air in to cool it down.  3MF & STL attached. 

**Print settings:**
- 0.16mm Layer height
- No bottom/top shell layers
- infill: Honeycomb , 25%

[LG-ESP-Case.zip](https://github.com/JanM321/esphome-lg-controller/files/14909019/LG-ESP-Case.zip)

![PXL_20240408_173224074](https://github.com/JanM321/esphome-lg-controller/assets/73469840/af112c9f-e25b-4b03-b1a7-70eac28c76f7)
![PXL_20240408_173248306 MP](https://github.com/JanM321/esphome-lg-controller/assets/73469840/8758ade7-4d21-479b-8c15-653fbb7975ef)
![PXL_20240408_173310965](https://github.com/JanM321/esphome-lg-controller/assets/73469840/471f42d1-51bc-4e53-bcee-989082523d1f)

**Comment by sergeipr** [comment]
This is awesome! Thank you! 🙏 

I don't own a printer and thought of ordering from [JLCPCB 3D print service](https://jlc3dp.com/), but ran into some difficulties. I will list them below and maybe try to remix your model (never did this before)
- Separate the multiple parts into separate files [see FAQ first question](https://jlc3dp.com/help/article/217-3D-Printing-FAQ)
- JLC doesn't provide top/bottom shell layer controls, so the vent holes need to be incorporated into the model

**Comment by mptr8956** [comment]
Hi, great to hear! 

Just FYI: it usually depends on layer size and calibration of the printer if the top cover sits well and secure on the bottom part, therefore you sometimes need to adjust the size of the top cover a little bit so that it sticks well and isn't loose. I did this already for my printer and in the files (shrinked it to 99.93% of the original size) and that works well.

If you order it online, be sure to have 3 or 4 different sizes of the top cover so you don't have to order again.

If you like to, I can upload the raw files for Fusion 360 so you can play around with them directly and insert some ventilation holes!


Btw where do you live? May be easier if I print one or two cases for you (if you don't need 50 🙈) and ship it!

**Reply by sergeipr** [reply]
Hi, thanks for the info! l need 4 pieces and live in the Middle East. Don't want to bother you, it's probably time to get a 3d printer 😀

**Reply by mptr8956** [reply]
that's the best idea anyway. once you have one, you'll need it! :D

**Comment by JanM321** [comment]
@mptr8956 I forgot to reply to this, but the pictures look great. I know nothing about 3d printers but if this is easy to reuse we could add the files to the repository (a `hardware-tiny/case/` or `hardware-tiny-case/` directory or something). Feel free to create a pull request if you're interested in this.

I could also put a link to this discussion in the README. Let me know what you think.
